src/html/listener.rs

```diff
@@ -17,6 +17,7 @@ macro_rules! impl_action {
 
             /// A wrapper for a callback.
             /// Listener extracted from here when attached.
+            #[allow(missing_debug_implementations)]
```

Comment:
I don't know about allowing a missing debug here, but I've never seen a `Wraper<_>` in application code.

Reply:
Yeah, this is fine. `Wrapper` is used for VTag listeners